src/AltinnCore/AltinnCore.Runtime.Db/Dockerfile

```diff
@@ -0,0 +1,19 @@
+#dotnet:2.1.300-sdk
+FROM microsoft/dotnet@sha256:d1ad61421f637a4fe6443f2ec204cca9fe10bf833c31adc6ce70a4f66406375e AS build
+WORKDIR /src/AltinnCore/AltinnCore.Runtime.Db
+
+# Copy csproj and restore as distinct layers
+COPY *.csproj ./
+
+# Copy everything else and build
+COPY . ./
+
+RUN dotnet build AltinnCore.Runtime.Db.csproj -c Release -o /app_output
```

Comment:
We are doing both build and publish in other Dockerfiles also, but do we need to do both?

Reply:
Keeping build until further notice.